### DISCUSSION TITLE
Still active?

### DISCUSSION THREAD

**Discussion by Sigurdur42** [israel-dryer/ttkbootstrap#576] [General]
Hi,
Is this framework still under active development? The latest release is more than a year old.
Thx

**Comment by zerootoad** [comment]
last commit was 7 months ago so i suppose its even if not its one of the libraries that work best with tk

![image](https://github.com/israel-dryer/ttkbootstrap/assets/153208404/49fa24e7-e744-4d80-839e-80302d492578)
ì

**Comment by rdbende** [comment]
> Is this framework still under active development?

No, it's clearly not

**Reply by hyperion4** [reply]
> > Is this framework still under active development?
> 
> No, it's clearly not

Well, it works fine and we don't have many choices out there for modern GUIs.
I think we have only ttkbootstrap and CustomTkinter?
No offense but your ttk themes are not an option for me.
My apps usually have many notebooks, treeviews and widgets and your themes make them so slow and laggy every time I've tried to use them.
For advanced apps our options are ttkbootstrap and CustomTkinter.
ttkbootstrap still works fine in apps with many widgets and it is super fast.

**Reply by meetgrinder** [reply]
I was just meandering around the web and wondering if something like this would work in pyodide as an interesting way to make desktop browser UIs.
